hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/dto/AttributeSearchRequestTest.kt

```diff
@@ -538,13 +538,16 @@ class AttributeSearchRequestTest(@Autowired val objectMapper: ObjectMapper) {
                      "attribute": "firstName",
                      "condition": "IS",
                      "searchTerm": "John"
-                   }
-                 ],
-                 "bodyPartMatchers": [
+                   },
+                   {
+                     "attribute": "tattoos.bodyPart",
+                     "condition": "IS",
+                     "searchTerm": "shoulder"
```

Comment:
Removed specific body part matcher - it's just a text matcher on a nested object